src/Activation/NotificationPermissions.spec.tsx

```diff
@@ -109,7 +109,7 @@ const createPermissionProviderValue = (
       request: requestPermission,
     },
     exposureNotifications: {
-      status: { authorized: true, enabled: true },
+      status: { authorized: true, enabled: true, authorizedAndEnabled: true },
```

Comment:
Does this introduce a risk of getting into impossible states? e.g. `{ authorized: false, enabled: false, authorizedAndEnabled: true }`.  If we need to dry up logic, what do you think about introducting a helper function or maybe updating the exposed status type altogether? would a union type be a better fit than an object?

Reply:
Oooh I like this idea. Going to noodle on it for a sec 🤔